packages/augur-ui/src/modules/common/form.tsx

```diff
@@ -940,17 +940,21 @@ export class ReportingRadioBar extends Component<ReportingRadioBarProps, {}> {
 
     if (scalar) {
       stake = {
-        preFilledStake: formatRep("0"),
-        bondSizeCurrent: formatRep("1"),
-        bondSizeCurrent: formatRep("1"),
-      }
+        outcome: null,
+        stakeCurrent: '0',
+        bondSizeCurrent: '1',
+        stakeRemaining: '1',
```

Comment:
These are hardcoded values, used for dev. Replace with actual values from getter

Reply:
Ok, will update later today.

Reply:
Actually, I think this is only used for the "Other Outcomes radio button":

<img width="678" alt="Screen Shot 2019-09-22 at 11 40 41 AM" src="https://user-images.githubusercontent.com/11491365/65392854-35103700-dd2e-11e9-8d60-9aec9edca739.png">

For these outcomes, isn't it ok to use hard-coded values since they won't have any stake on them yet?

Reply:
Never mind, I think you mean for the bondSizeCurrent/stakeRemaining, not the stakeCurrent. I think this will require adding a new prop for ReportingRadioBar?

Reply:
Really we need to use StakeDetails and not have the UI create its own internal object and pass it around.  These UI components were created before the new StakeDetails properties rename

Reply:
Also the UI dispute components will need to know bond size new outcome value that is on the parent object of stakeDetails. Its used when an outcome hasnt been staked on.

Reply:
Isn't this info already in state.marketInfos[marketId].disputeInfo?

Reply:
Yes thats what I’m referring to. Better if I would have said use the same types from the getMarkets/getMarketsInfo getters.

Reply:
For example, sdk doesnt diff between pre-filled and stakeCurrent. imo, the UI shouldnt in its variables. We do in the viz components.

Reply:
Ok, seems like ModalReporting should be passing in the disputeInfo for the market into RadioBarGroup as a prop, which should then be passed around similarly from there?

Reply:
Phoebe and talked about how the components are put together. Really containers should be used and only minimum props are passed. Can you organize the components? Good op to learn child prop stuff and containers.

Reply:
Sure, I can do that. We have plans today, so I probably won't be able to finish by tomorrow, but I'll start working on that later today.